views/fightScreen.py

```diff
@@ -0,0 +1,622 @@
+        self.mainLayout.addWidget(self.player1Magic_Lbl, row, innerCol)
+
+        self.player1_magicMeter = self.player1_meters["magic"]
+        self.mainLayout.addWidget(self.player1_magicMeter, row, innerCol + 1, 1, 2)
+
+        self.player2Magic_Lbl = QLabel("Magic Meter: " + str(self.P2MagicMeter))
+        self.player2Magic_Lbl.setStyleSheet("color: white;")
+        self.mainLayout.addWidget(self.player2Magic_Lbl, row, rightCol)
+
+        self.player2_magicMeter = self.player2_meters["magic"]
+        self.mainLayout.addWidget(self.player2_magicMeter, row, rightCol + 1, 1, 2)
+        row += 1
+
+        self.mainLayout.addItem(
+            QSpacerItem(0, 10, QSizePolicy.Fixed, QSizePolicy.Fixed), row, innerCol
+        )
+        row += 1
+
+        self.player1Mech_Lbl = QLabel("Mechanical Meter: " + str(self.P1MechMeter))
+        self.player1Mech_Lbl.setStyleSheet("color: white;")
+        self.mainLayout.addWidget(self.player1Mech_Lbl, row, innerCol)
+
+        self.player1_mechMeter = self.player1_meters["mechanical"]
+        self.mainLayout.addWidget(self.player1_mechMeter, row, innerCol + 1, 1, 2)
+
+        self.player2Mech_Lbl = QLabel("Mechanical Meter: " + str(self.P2MechMeter))
+        self.player2Mech_Lbl.setStyleSheet("color: white;")
+        self.mainLayout.addWidget(self.player2Mech_Lbl, row, rightCol)
+
+        self.player2_mechMeter = self.player2_meters["mechanical"]
+        self.mainLayout.addWidget(self.player2_mechMeter, row, rightCol + 1, 1, 2)
+        row += 1
+
+        self.mainLayout.addItem(
+            QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.Fixed), row, innerCol
+        )
+        row += 1
+
+        self.player1_Pic = QLabel("")
+        self.player1_Pic.setAlignment(Qt.AlignCenter)
+        self.player1_Pic.setStyleSheet("min-width: 380px;")
+        self.player1_Pic.setPixmap(QPixmap("./assets/ready.jpg"))
+        self.mainLayout.addWidget(self.player1_Pic, row, innerCol, 1, 3)
+
+        self.mainLayout.addItem(
+            QSpacerItem(30, 0, QSizePolicy.Fixed, QSizePolicy.Fixed), row, innerCol + 3
+        )
+
+        self.player2_Pic = QLabel("")
+        self.player2_Pic.setAlignment(Qt.AlignCenter)
+        self.player2_Pic.setStyleSheet("min-width: 380px;")
+        self.player2_Pic.setPixmap(QPixmap("./assets/ready.jpg"))
+        self.mainLayout.addWidget(self.player2_Pic, row, rightCol, 1, 3)
+        row += 1
+
+        self.mainLayout.addItem(
+            QSpacerItem(0, 20, QSizePolicy.Fixed, QSizePolicy.Fixed), row, innerCol
+        )
+        row += 1
+
+        self.P1Weapon_Pic = QLabel("")
+        self.P1Weapon_Pic.setPixmap(
+            QPixmap("./assets/sword.jpg").scaledToWidth(50)
+        )
+        self.mainLayout.addWidget(self.P1Weapon_Pic, row, innerCol)
+
+        self.P1Defense_Pic = QLabel("")
+        self.P1Defense_Pic.setPixmap(
+            QPixmap("./assets/shield.jpg").scaledToWidth(50)
+        )
```

Comment:
Let's make these dynamic, so they won't be a pain to edit in the future. I.e. loop over `self.imageAssets`

Reply:
Definitely. Add to issue list.